lib/jekyll-github-metadata/ghp_metadata_generator.rb

```diff
@@ -9,11 +9,17 @@ class GHPMetadataGenerator < Jekyll::Generator
       attr_reader :site
 
       def generate(site)
+        if defined?(@_already_generated)
```

Comment:
`after_init` was introduced only a few versions ago so this would stop working for older versions. Probably fine, but just calling that out.

In the GitHub Pages gem, we call the configuration reset in `after_reset`, which is called towards the top. Does that seem OK?